conn.go

```diff
@@ -25,16 +22,11 @@ type Conn struct {
 }
 
 // NewConn create wrapper on net.Conn.
-func NewConn(
-	conn net.Conn,
-	logger Logger,
-	headerParser HeaderParser,
-	trustedAddr bool,
-) net.Conn {
+func NewConn(conn net.Conn, logger Logger, headerParser HeaderParser, trustedAddr bool) net.Conn {
```

Comment:
I don’t understand why multiline function declarations have been replaced with single-line ones. This makes reading difficult.

Reply:
It depends on the writing habits: it is quite rare in Go to do this, but this is common in Ruby or JS.